build.sbt

```diff
@@ -56,11 +53,28 @@ lazy val microservice = Project(appName, file("."))
     )
   )
 
-commands ++= Seq(
-  Command.command("run-all-tests") { state => "test" :: "it:test" :: state },
 
-  Command.command("clean-and-test") { state => "clean" :: "compile" :: "run-all-tests" :: state },
+lazy val it = (project in file("it"))
+  .enablePlugins(PlayScala)
+  .dependsOn(microservice % "test->test")
+  .settings(DefaultBuildSettings.itSettings())
+  .settings(
+    name := "integration-tests",
+    Test / testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-eT"),
+    headerSettings(Test) ++ automateHeaderSettings(Test)
```

Comment:
I think you might need a bloop setting here, which has been included in other repositories.
```  inConfig(Test)(BloopDefaults.configSettings),```

Reply:
Comment withdrawn after discussion